src/getter.js

```diff
@@ -9,6 +11,19 @@ import { isKeyPath } from './key-path'
  */
 const identity = (x) => x
 
+class GetterClass {
```

Comment:
I think this would be better as a pseudo-constructor.  IE I dont like the idea of changing the Getter type because now we have to handle both a `Getter` instance and a plain array getter in every interface.

Instead `Getter(['items'])` returning simply `['items']` makes a lot of sense to me for backwards compat and simplicity.